jboss-kie-kieserver/tests/bats/jboss-kie-kieserver.bats

```diff
@@ -617,3 +620,27 @@ teardown() {
     echo "GC_MAX_METASPACE_SIZE=${GC_MAX_METASPACE_SIZE}"
     [[ "${GC_MAX_METASPACE_SIZE}" == "${KIE_SERVER_MAX_METASPACE_SIZE}" ]]
 }
+
+@test "Verify if the jbpm cache is contained in the standalone-openshift.xml when KIE_SERVER_JBPM_CLUSTER is true" {
+  export KIE_SERVER_JBPM_CLUSTER="true"
+  export KIE_SERVER_JBPM_CLUSTER_TRANSPORT_LOCK_TIMEOUT="60000"
```

Comment:
can you remove it from here to test the default value and add a new one to test a value different than the default?

Reply:
Done https://github.com/jboss-container-images/jboss-kie-modules/pull/434/files#diff-c6b09ca9c95a186444fb9d39db3c31baee18124133826800ccedc74a82646bf2R626